### ISSUE TITLE
Add text-regression tasks

### ISSUE THREAD

**Issue by adrinjalali** [skops-dev/skops#102]
Similar to #100, but we should also make sure the config file and the README contain the right components for the widget to work.

This might also involve adding a text-classification alias on the api-inference-community side.

**Comment by BenjaminBossan** [comment]
Once those are done, we should an entry for sklearn [here](https://huggingface.co/docs/hub/models-libraries).

**Comment by adrinjalali** [comment]
That list seems quite misleading, as in, it doesn't seem to show what tasks are supported by each library, so we could already add ours there. But I'm in no rush.

**Comment by BenjaminBossan** [comment]
Yeah, no rush, but it seems that with those tasks finished, we have covered the most important ones for sklearn.

**Comment by adrinjalali** [comment]
Renamed since we now support text-classification.

**Comment by adrinjalali** [comment]
Closing as `hub_utils` is deprecated away.
